jest.config.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     '^browser-components(.*)$': '<rootDir>/src/browser/components$1',
     '^browser-services(.*)$': '<rootDir>/src/browser/services$1',
     '^browser-hooks(.*)$': '<rootDir>/src/browser/hooks$1',
+    // '^common$': '<rootDir>/src/common', // Not needed???
```

Comment:
I think it's not needed since the folder name is the same as the alias, so it'll find it automatically. Not true for 'graph-visualization' and 'graphVisualization' though, so that's why it's in there